src/Symbolics.jl

```diff
@@ -197,6 +199,18 @@ for sType in [Pair, Vector, Dict]
     @eval substitute(expr::Arr, s::$sType; kw...) = wrap(substituter(s)(unwrap(expr); kw...))
 end
 
+# Symbolic solver
+include("./solver/preprocess.jl")
```

Comment:
How does this relate to 

```julia
include("solver.jl")
export solve_single_eq
export solve_system_eq
export lambertw
```

above?

Reply:
if you're referring to preprocess.jl, no relation whatsoever. If you're referring to the actual solver, there is one case that that algebraic solver handles which we dont.
```julia
    #lambert w
    @test correctAns(solve_single_eq(x^x~2,x),[log(2.0)/lambertw(log(2.0))])
    @test correctAns(solve_single_eq(2*x*exp(x)~3,x),[LambertW.lambertw(3.0/2.0)])
```